includes/class-wc-gateway-stripe.php

```diff
@@ -411,7 +411,7 @@ public function admin_scripts() {
 				__( '<strong>Warning:</strong> your site\'s time does not match the time on your browser and may be incorrect. Some payment methods depend on webhook verification and verifying webhooks with a signing secret depends on your site\'s time being correct, so please check your site\'s time before setting a webhook secret. You may need to contact your site\'s hosting provider to correct the site\'s time.', 'woocommerce-gateway-stripe' ),
 				[ 'strong' => [] ]
 			),
-			'upe_setting_value' => $gateway_settings['upe_checkout_experience_enabled'] ?? 'no',
+			'upe_setting_value' => isset( $gateway_settings['upe_checkout_experience_enabled'] ) ? 'no' : $gateway_settings['upe_checkout_experience_enabled'],
```

Comment:
I'm curious about this change as I thought `??` was essentially chaining an `isset()` call with a ternary which is what you've done here. Is this being done for PHP < 7.0 compatibility?

Reply:
Yes!